doc/netplan.md

```diff
@@ -831,6 +831,14 @@ Example:
 
      **Requires feature: eswitch-mode**
 
+``infiniband-mode`` (scalar) – since **0.105**
+
+:    (InfiniBand devices only) Change the operational mode of a IPoIB device.
+     Possible values are ``datagram`` or ``connected``, if unspecified the
+     kernel's default configuration is used.
```

Comment:
this is a run-on sentence, should say: "Possible values are ``datagram`` or ``connected``.  If unspecified [...]"